nixos/modules/services/misc/paperless.nix

```diff
@@ -439,5 +489,37 @@ in
         gid = config.ids.gids.paperless;
       };
     };
-  };
+  }) (lib.mkIf (cfg.enable && cfg.exporter.enable) {
+
+    systemd.tmpfiles.rules = [
+      "d '${cfg.exporter.directory}' - ${cfg.user} ${config.users.users.${cfg.user}.group} - -"
+    ];
+
+    systemd.timers.paperless-exporter = lib.mkIf (cfg.exporter.onCalendar != null) {
+      description = "paperless exporter timer";
+      wantedBy = [ "timers.target" ];
+      timerConfig = {
+        OnCalendar = cfg.exporter.onCalendar;
+        AccuracySec = "5m";
+      };
+    };
+
+    systemd.services.paperless-exporter = {
```

Comment:
Since the discussion here, I actually found a somewhat clean method to make the exporter shut down paperless while it runs:

https://github.com/Atemu/nixos-config/blob/5323b915a63748c7ffa891f813a753fa16601df5/modules/paperless/module.nix#L62-L69

(See also https://github.com/systemd/systemd/issues/34954)

Reply:
Sounds good. However, this is now constantly causing those messages when the triggers happen:

```
simple # [  129.600073] systemd[1]: paperless-consumer.service: multiple trigger source candidates for exit status propagation (paperless-exporter.service, paperless-exporter.service), skipping.
```

Ideas?

Reply:
None, that's the best we can do. See the linked systemd issue. That warning doesn't appear to do any harm.